librz/core/cmd/cmd_search.c

```diff
@@ -1138,7 +1186,7 @@ static RzList /*<RzCoreAsmHit *>*/ *construct_rop_gadget(RzCore *core, ut64 addr
 	return hitlist;
 }
 
-static void print_rop(RzCore *core, RzList /*<RzCoreAsmHit *>*/ *hitlist, PJ *pj, int mode) {
+static void print_rop(RzCore *core, RzList /*<RzCoreAsmHit *>*/ *hitlist, PJ *pj, RzOutputMode mode) {
```

Comment:
I recommend passing the whole `RzCmdStateOutput` to this function instead, especially important for adding the TABLE mode.